flow/project.py

```diff
@@ -314,7 +314,7 @@ class FlowCondition(object):
     :param callback:
         A function with one positional argument (the job)
     :type callback:
-        :py:class:`~signac.contrib.job.Job`
+        function(:py:class:`~signac.contrib.job.Job`)
```

Comment:
Our standard is to refer to these as callables (to allow for e.g. functors as well).